devops/windows/base-docker-wsl/conf/env/conf-env-low-1-pc.template

```diff
@@ -0,0 +1,105 @@
+export SPEC_SETTINGS=low
```

Comment:
Similar comments regarding java app from above. Do we need the settings them?

Reply:
Keep this --- high end can for example be single kafka broker or multiple kafka brokers      even for single node WSL run time environements